app/views/articles/_full_comment_area.html.erb

```diff
@@ -3,7 +3,7 @@
     <% if @article.show_comments %>
       <header class="relative flex justify-between items-center">
         <h3 class="fs-2xl fw-bold">Discussion</h3>
-        <div id="comment-subscription"><span class="crayons-btn crayons-btn--outlined">Subscribe</span></div>
```

Comment:
It was supposed to be a placeholder for this button until it's loaded so we don't have weird jump there. Is that neecessary to remove it?

Reply:
Well on my local I see two buttons. But it doesn't appear to be happening in production. Just testing locally again...

Reply:
Ahh, it looks like the Preact upgrade isn't in production yet. That's why.

Reply:
We'll merge this and I'm investigating the fix to bring the SSR button back.

Reply:
All sorted out in this commit, https://github.com/thepracticaldev/dev.to/pull/8768/commits/e6ad10106c19310e09185eb2a0dfc6aa0d8e4b8c